frontend/src/pages/profile/components/user-articles-genres-stats/libs/constants/constants.ts

```diff
@@ -0,0 +1,35 @@
+const GENRES_CHART_COLORS = [
+  '#44996b',
+  '#58a77c',
+  '#6bb588',
+  '#7cc392',
+  '#8fd19c',
+  '#a1e0a6',
+  '#b4efb1',
+  '#c6fdc1',
+  '#d9fecb',
+  '#ecffd6',
+  '#d6ecff',
+];
+
+const MAX_VISIBLE_GENRES = 10;
+
+const GENRES_CHART_SIZE = 500;
+
+const GENRES_CHART_INNER_RADIUS = 60;
+
+const GENRES_CHART_OUTER_RADIUS = 120;
+
+const OTHER_GENRES_KEY = 'others';
+
+const OTHER_GENRES_NAME = 'Others';
```

Comment:
Could be grouped to enum
```suggestion
const OtherGenre = {
  KEY: 'others',
  NAME: 'Others'
} as const;
```